dev.html

```diff
@@ -35,7 +36,22 @@
         bosh_service_url: 'https://conversejs.org/http-bind/', // Please use this connection manager only for testing purposes
         message_archiving: 'always',
         debug: true
-    });
+    }).then(function () {
+        setTimeout(function () {
+            // First we get the viewport height and we multiple it by 1% to get a value for a vh unit
+            const vh = window.innerHeight * 0.01;
+            // Then we set the value in the --vh custom property to the root of the document
+            document.documentElement.style.setProperty('--vh', `${vh}px`);
+            window.addEventListener('resize', () => {
+            // We execute the same script as before
+            const vh = window.innerHeight * 0.01;
+            document.documentElement.style.setProperty('--vh', `${vh}px`);
+            });
+        }, 1000);
```

Comment:
This code needs to go into one of the core modules so that it's always available. Otherwise you'd need to remember to always add this when you call `_converse.initialize`.

Also, instead of using `setTimeout`, we should rather listen for an event that signifies that the viewport height is calculable.

The most low-level view module is converse-chatboxviews.js.

I think you can register an event handler for `chatBoxViewsInitialized` and then in that handler add the above code.

Also, you still need to add an event handler for the `resize` event so that the `vh` value gets recalculated when the viewport changes size.
See here:  https://css-tricks.com/the-trick-to-viewport-units-on-mobile/#article-header-id-2